app/models/review.rb

```diff
@@ -1,6 +1,29 @@
 class Review < ApplicationRecord
   validates :overall_rating, presence: true, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 5 }
```

Comment:
FYI numericality will by default validate presence, so you don't technically need `presence:true` here